src/linking/remove_internal_symbols.cpp

```diff
@@ -130,6 +154,9 @@ void remove_internal_symbols(
   special.insert("__placement_new_array");
   special.insert("__delete");
   special.insert("__delete_array");
+  // plus any extra symbols we wish to keep
+  if(!keep.empty())
```

Comment:
I don't think that test even is necessary for `.empty()` is the same as `.begin() == .end()`.

Reply:
fixed